### PR TITLE
support jetty server work thread pool metric monitor

### PR DESCRIPTION
**Pull request by xu1009**
- [x] If this pull request closes/resolves/fixes an existing issue, replace the issue number. https://github.com/apache/skywalking/issues/10736.
- [x] Update the [`CHANGES` log](https://github.com/apache/skywalking-java/blob/main/CHANGES.md).

### CODE REVIEW

**Comment by xu1009** [comment]
the plugin supports jetty 11，but jetty 11 example is different with lower version, should i write new scenario test for 11

**Comment by xu1009** [comment]
ok，i  write another test for 11, have fun

File: .github/workflows/plugins-jdk17-test.0.yaml [hunk 1]
```diff
@@ -57,6 +57,7 @@ jobs:
         case:
           - jdk17-with-gson-scenario
           - resttemplate-6.x-scenario
+          - jetty-thread-pool-scenario
```
Comment (wu-sheng) [inline .github/workflows/plugins-jdk17-test.0.yaml:60]:
```suggestion
          - jetty-thread-pool-scenario
          - jetty-11.x-thread-pool-scenario
```

Comment (wu-sheng) [reply .github/workflows/plugins-jdk17-test.0.yaml:60]: @xu1009 I believe the both cases should be added.

**Review by wu-sheng — APPROVED** [review]

**Pull request merged**